src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,85 @@
-import React from 'react';
+import classNames from 'classnames';
+
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
+import { MovieData } from '../../types/MovieData';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+const imdbLink = 'https://www.imdb.com/title';
+const defaultImg = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+type Props = {
+  setFoundMovie: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ setFoundMovie }) => {
+  const [isError, setIsError] = useState(false);
+  const [searchInputValue, setSearchInputValue] = useState('');
+  const [movieFromServer, setMovieFromServer] = useState<
+  MovieData | ResponseError | null>(null);
```

Comment:
we can create another state for error message, do not mix movie and error in one state